instantmenu.c

```diff
@@ -1422,8 +1427,15 @@ setup(void)
 	int a, di, n, area = 0;
 #endif
 	/* init appearance */
-	for (j = 0; j < SchemeLast; j++)
-		scheme[j] = drw_scm_create(drw, colors[j], 3);
+ 	for (j = 0; j < SchemeLast; j++) {
+		scheme[j] = drw_scm_create(drw, (const char**)colors[j], 2);
```

Comment:
```suggestion
		scheme[j] = drw_scm_create(drw, (const char**)colors[j], 3);
```
Like @uvera suggested.